### ISSUE TITLE
Right hand bar body samples can't be exanded/collapsed

### ISSUE THREAD

**Issue by lornajane** [Redocly/redoc#1342]
![Screenshot from 2020-08-05 20-30-33](https://user-images.githubusercontent.com/172607/89455795-c0ce5e80-d75a-11ea-8b68-bd3348098bd7.png)

The plus/minus buttons are not links or buttons, they are something called `div.collapser`, and it means that my accessibility tooling cannot interact with them (I've only recently spotted they are there!)
